state/execution.go

```diff
@@ -247,6 +270,10 @@ func (blockExec *BlockExecutor) Commit(
 		TxPreCheck(state),
 		TxPostCheck(state),
 	)
+	if err != nil {
```

Comment:
This was not checked before. Any specific concern behind adding it?

Reply:
UPDATE: discussed with Barry. Agreed to look into the issue in more detail. Barry will try to figure out who wrote that and why.

Reply:
[Issue in collab repo.](https://github.com/informalsystems/audit-skip/issues/9)

Reply:
Fixed in https://github.com/skip-mev/mev-tendermint/pull/126